modules/oe_theme_content_consultation/config/install/core.entity_view_display.node.oe_consultation.full.yml

```diff
@@ -0,0 +1,281 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.full
+    - field.field.node.oe_consultation.oe_consultation_additional_info
+    - field.field.node.oe_consultation.oe_consultation_aim
+    - field.field.node.oe_consultation.oe_consultation_closed_text
+    - field.field.node.oe_consultation.oe_consultation_contacts
+    - field.field.node.oe_consultation.oe_consultation_deadline
+    - field.field.node.oe_consultation.oe_consultation_guidelines
+    - field.field.node.oe_consultation.oe_consultation_legal_info
+    - field.field.node.oe_consultation.oe_consultation_opening_date
+    - field.field.node.oe_consultation.oe_consultation_outcome
+    - field.field.node.oe_consultation.oe_consultation_outcome_files
+    - field.field.node.oe_consultation.oe_consultation_response_button
+    - field.field.node.oe_consultation.oe_consultation_target_audience
+    - field.field.node.oe_consultation.oe_departments
+    - field.field.node.oe_consultation.oe_subject
+    - field.field.node.oe_consultation.oe_summary
+    - field.field.node.oe_consultation.oe_teaser
+    - node.type.oe_consultation
+  module:
+    - datetime
+    - field_group
+    - link
+    - rdf_skos
+    - text
+    - user
```

Comment:
We need to make sure that these dependencies are also dependencies of the module, also for the other exported config. Of course not for the user module. This needs to be reflected in the readme, in case such dependencies are not in Drupal core.